nifi-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -2003,6 +2003,31 @@ one of the nodes, and the User Interface should look similar to the following:
 
 image:ncm.png["Clustered User Interface"]
 
+[[cluster_firewall_configuration]]
+=== Cluster Firewall Configuration
+
+NiFi clustering supports network access restrictions using a custom firewall configuration.
+The `nifi.cluster.firewall.file` property can be configured with a path to a file containing hostnames, IP addresses, or
+subnets of permitted nodes. The Cluster Coordinator uses the configuration to determine whether to accept or reject
+heartbeats and connection requests from potential cluster members.
+
+The configuration file format expects one entry per line and ignores lines beginning with the `#` character. NiFi uses
+standard hostname resolution to convert names to IP addresses. The configuration file supports IPv4 addresses or subnet
```

Comment:
Could you add more info on what "standard hostname resolution" means? For example, I believe NiFi has no inherent ability to resolve an IP address. This capability relies on that of the host.

Reply:
What do you think about changing it to read as follows?

NiFi uses standard Java host name resolution to convert names to IP addresses. The Java host name resolution leverages a combination of local machine configuration and network services, such as DNS.

Reply:
Yes, that looks good. Thanks.

Reply:
Thanks, I appreciate the feedback!